timesketch/frontend-ng/src/components/LeftPanel/AnalyserResults.vue

```diff
@@ -79,18 +96,27 @@ export default {
   },
   data: function () {
     return {
-      analyzerResults: [],
       expanded: false,
+      analyzerResultsReady: false,
+      analyzerResults: [],
+      search: '',
     }
   },
+  // TODO: Have an automatic poll for new analyzers running when they are triggered.
   computed: {
     sketch() {
       return this.$store.state.sketch
     },
     meta() {
       return this.$store.state.meta
     },
+    // sketchAnalyzerList() {
+    //   return this.$store.state.sketchAnalyzerList
+    // },
```

Comment:
remove?

Reply:
Since I moved the collection of analyzer results back into the component, I need it again :)